library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -65,6 +65,9 @@ class FsSnapshot
     LIST_SNAPSHOTS_CMD = "LANG=en_US.UTF-8 /usr/bin/snapper --no-dbus --root=%{root} list"
     VALID_LINE_REGEX = /\A\w+\s+\| \d+/
 
+    # Predefined snapshot cleanup strategies (the user can define custom ones, too)
```

Comment:
Not in the context of this Ruby class, but in general, he can. Snapper comes with those predefined ones you see here, but a user can write his own scripts and make snapper use them on request. Arvin knows more details about that.